content/app/development/configuration/datafields/_index.en.md

```diff
@@ -41,26 +40,26 @@ Konfigurasjonen til en app med to definerte datafelter vil se slik ut:
     }]
   ```
 
-Resultatet vil være en liste på instansobjektet med verdier fra de konfigurerte feltete:
+The result will be a list in the instance object with values from the configured fields:
 ```json
 "dataValues": {
     "AnsettelseAntAar": 10,
     "Navn": "Ola Nordmann"
 }
 ```
-Legg merke til at det på instansobjektet heter `dataValues` mens når det konfigureres heter `dataFields`, det er fordi 
-`dataValues`er resultatet av konfigureringen som gjøres på `dataFields`.
+Notice that the instance object is named `dataValues` even when the configured is named `dataFields`, this is because `dataValues` is the 
+result of the configuration which is done in `dataFields`.
 
-## Manuelt
-For å legge til dataverdier manuelt benyttes metoden `UpdateDataValues` fra IInstance interfacet. Det er den samme metoden som kalles når dataverdier populeres fra konfigurasjon og den sørger for å flette sammen verdiene til en liste. 
+## Manually
+To manually add datavalues the method `UpdateDataValues`from the IInstance interface is used. It is the same method which is called when datavalues are populated from configuration and it ensures seemless gathering of values into a list. 
```

Comment:
```suggestion
To manually add data values the method `UpdateDataValues`from the IInstance interface is used. It is the same method which is called when `dataValues` are populated from configuration and it will merge all values into a list.
```

Reply:
> 

```suggestion
To manually add data values the method `UpdateDataValues`from the IInstance interface is used. It is the same method which is called when the fields in `dataValues` are populated from configuration and it will merge all values into a list. 
```